lms/templates/shoppingcart/registration_code_receipt.html

```diff
@@ -21,7 +20,7 @@ <h1 class="title">
         <img class="item-image" src="${course_image_url(course)}"
             alt="${_("{course_number} {course_title} Cover Image").format(
               course_number=course.display_number_with_default,
-              course_title=get_course_about_section(request, course, 'title'),
+              course_title=course.display_name_with_default,
```

Comment:
This is a tricky one to fix right now without introducing double-escaping.  Since it is just course_number, we'll leave this alone until display_name_with_default gets fixed.